### DISCUSSION TITLE
Do you know the way how to integrate App for updating variant inventory?

### DISCUSSION THREAD

**Discussion by annaoutlook** [Shopify/dawn#3153] [Q&A] [answered]
Customers required to implement the function for updating variant inventory using command line.
How do you carry out this task?

**Comment by top-comengineer** [comment]
I can resolve this issue as following:
1. export current inventory to .csv file.
2. develope app using python which could update inventory of variant by coding command.

**Reply by annaoutlook** [reply, marked as answer]
thanks.
